app.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const cors =  require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const port = 3000;
+const school = require("./School.js")
+
```

Comment:
Here you would need a new instance of the class you're going to work with:

`let newSchool = new School();`